tests/mir-opt/inline/unchecked_shifts.unchecked_shl_unsigned_smaller.Inline.panic-unwind.diff

```diff
@@ -23,18 +26,30 @@
           _4 = copy _2;
 -         _0 = core::num::<impl u16>::unchecked_shl(move _3, move _4) -> [return: bb1, unwind continue];
 +         StorageLive(_5);
++         StorageLive(_8);
++         StorageLive(_9);
 +         StorageLive(_6);
-+         _6 = UbChecks();
-+         switchInt(copy _6) -> [0: bb2, otherwise: bb1];
++         StorageLive(_7);
++         _7 = &_4;
++         _6 = {closure@$SRC_DIR/core/src/num/uint_macros.rs:LL:COL} { 0: copy _7 };
++         StorageDead(_7);
++         _5 = contract_check_requires::<{closure@core::num::<impl u16>::unchecked_shl::{closure#0}}>(move _6) -> [return: bb1, unwind continue];
```

Comment:
All these extra checks would slow down debug builds a lot in terms of runtime performance and likely a bit in build times, right? Can we check by how much exactly?